.travis.yml

```diff
@@ -1,17 +1,14 @@
 language: node_js
 sudo: false
 node_js:
-  - "11.4.0"
+  - '11.4.0'
 
 cache: yarn
 
 install:
   - yarn install
```

Comment:
The source of the discrepancy between circle CI and Travis is probably the fact that travis is not using the lockfile like Circle CI is doing. Circle CI has this command:
`yarn install --frozen-lockfile --non-interactive`
We should use this here to make both consistent.

Reply:
done